manifest-list.md

```diff
@@ -64,6 +66,13 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 
         This OPTIONAL property specifies an array of strings, each specifying a mandatory CPU feature (for example `sse4` or `aes`).
 
+- **`elements`** *array of objects*
+
+  This OPTIONAL property contains a list of non-manifest objects.
```

Comment:
So, the `manifests` array is intended for `application/vnd.oci.image.manifest.v1+json` and `application/vnd.oci.image.manifest.list.v1+json`. Any distinguishing of non-OCI manifest lists ... I'm not real sure. This field is also what Docker uses presently for their manifest-list and therefore needs some backwards-compat consideration. 

This `elements` (or whatever) is intended for additional metadata like AppStream or other relevant repo-metadata (say XML or sqlite objects) that are facilitating this image-layout. 

I would think it much cleaner to just have a single field like `descriptors` where they all live. Then this desire to distinguish goes away.